core/eval.go

```diff
@@ -23,6 +23,7 @@ func init() {
 	txnCommands = map[string]bool{"EXEC": true, "DISCARD": true}
 }
 
+// Returns the status of the connection with a "PONG"
```

Comment:
"returns the status of the connection" seems wrong. We are simply responding with `PONG`. Let's change the language here.

Reply:
updated.